### PR TITLE
Publish script

### PR DESCRIPTION
**Pull request by aaronnorby**

### CODE REVIEW

File: scripts/publish.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+const errColor = chalk.bold.red;
+const goodColor = chalk.bold.green;
+
+const buildCommand = 'npm run build';
+
+inquirer.prompt([
+  {
+    type: 'list',
+    name: 'publish',
+    message: 'Are you sure you want to publish? This will run all tests and build first.',
```
Comment (aschiopu) [inline scripts/publish.js]: Shouldn't this say "Select publish version?"

File: scripts/publish.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    console.log(errColor('aborting publish'));
+    return;
+  }
+
+  const execOpts = {
+    stdio: [0, 1, 2]
+  };
+
+  //run tests
+  console.log(errColor('no tests are configured.😢 someone should fix that.'));
```
Comment (aschiopu) [inline scripts/publish.js]: lol

File: scripts/publish.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    console.log(runColor('publishing to npm'));
+    execSync('npm publish', execOpts);
+  } catch (e) {
+    console.log(errColor('Publishing failed'));
+    return;
+  }
+
+  try {
+    console.log(runColor('running git push origin master'));
+    execSync('git push origin master', execOpts);
```
Comment (aschiopu) [inline scripts/publish.js]: we should check we are on master branch as well right?

Comment (aaronnorby) [inline scripts/publish.js]: good point

**Pull request merged**